guides/getting_started.md

```diff
@@ -2,27 +2,17 @@
 
 **Prerequisite**: Gradio requires Python 3.7 or above, that's it! 
 
-### Introduction
+### What Problem is Gradio Solving? 😲
 
-Gradio allows you to easily build machine learning demos, data science dashboards, or other kinds of web apps, **entirely in Python**. These web apps can be launched from wherever you use Python (jupyter notebooks, colab notebooks, Python terminal, etc.) and shared with anyone instantly using Gradio's auto-generated share links.
+One of the *best ways to share* your machine learning model, API, or data science workflow with others is to create an **interactive demo** that allows your users or colleagues to try out the demo in their browsers. 
 
-To offer both simplicity and more powerful and flexible control for advanced web apps, Gradio offers two different APIs to users:
+A web-based demo is great as it allows anyone who can use a browser (not just technical people) to intuitively try their own inputs and understand what you've built. 
 
-* `gradio.Interface`: a high-level API that allows you to create a machine learning demo simply by specifying a list of inputs and outputs. It often takes just a few lines of Python to create a fully usable and share machine learning demo.
+However, creating such web-based demos has traditionally been difficult, as you needed to know backend frameworks (like Flask or Django) for serving the web app, containerization tools (like Docker) to package the model, databases to store data or users, and front end web development (HTML, CSS, JavaScript) to build a GUI for your demo. 
```

Comment:
Hmm, I kinda feel like this is a bit long, WDYT @aliabd and @aliabid94?

Reply:
Yes, feel the same way. Working on shortening it

Reply:
We could just say `without any web development or hosting`